backend/src/case_management/case_file_action/case_file_action.service.ts

```diff
@@ -121,13 +121,17 @@ export class CaseFileActionService {
   }
 
   //Used to return the all the actions of a given type for a specific case
-  async findActionsByCaseIdAndType(caseId: string, actionTypeCode: string) {
+  async findActionsByCaseIdAndType(caseId: string, actionTypeCodes: string | string[]) {
```

Comment:
At first I didn't like this and was going to suggest just taking in a string array and pushing any refactoring further back to pass an array of one item, but.. I've changed my mind and I think I like it 🤔